app.ts

```diff
@@ -0,0 +1,130 @@
+"use strict";
+window.onload = () => {onloadValues() };
+
+const rowHeight = 56;
+let from = 0;
+let to = 0;
+
+
+function amountOfRowsThatFit(){
+	let $buttonEl = $("button");
+
+	let windowHeigt = window.innerHeight;
+	let buttonDivheigth = $buttonEl.innerHeight;
+	let adaptedHeight = windowHeigt-buttonDivheigth;
+
+	return (adaptedHeight/rowHeight)-1;
+
+}
+
+
+function onloadValues(){
+	let $to = $("toValue");
+	let $from = $("toValue");
+	callHeaders();
+	from = 0;
+	let rowsFittable =amountOfRowsThatFit();
+	to = from+rowsFittable;
+
+}
+
+
+function pressedNext(){
```

Comment:
You need to sort out formatting throughout this PR and you should use a class or classes and methods as opposed to just functions. And when you create functions within methods use arrow functions, i.e. `() => {...}`